backend/src/org/commcare/session/SessionInstanceBuilder.java

```diff
@@ -39,7 +39,8 @@ private static void addSessionNavData(TreeElement sessionRoot, SessionFrame fram
      */
     private static void addDatums(TreeElement sessionData, SessionFrame frame) {
         for (StackFrameStep step : frame.getSteps()) {
-            if (SessionFrame.STATE_DATUM_VAL.equals(step.getType())) {
+            if (SessionFrame.STATE_DATUM_VAL.equals(step.getType()) ||
```

Comment:
but the type you added here was `STATE_DATUM_COMPUTED`, which isn't the new type. I understand the addition you made above now, but shouldn't this one be the same as that?